CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
Happy to discuss it and find the most suitable approach. The idea of different branches was inspired by [Apache Kafka repository](https://github.com/apache/kafka)